### DISCUSSION TITLE
My termux is conflicting with termux api

### DISCUSSION THREAD

**Discussion by psynyde** [termux/termux-app#2898] [Q&A] [answered]
I just flashed a12L rom (cherish os) on my device but before that i had termux installed with termux api as well.. but now termux seems to conflict with termux api. so how can i fix this..
### Preview:
![Screenshot_20220717-232626_Package installer](https://user-images.githubusercontent.com/75927811/179702171-b778355c-5a62-439e-9bbb-b017eb0a2ff5.png)

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app#installation
